README.md

```diff
@@ -182,6 +182,28 @@ This example will override the default configuration to:
 - add metric labels with values obtained from `app.kubernetes.io/name` Pod labels
 - rescan workload images with an interval of 24 hours
 
+#### Configure Trivy scan jobs
+
+A workload container image is scanned by scheduling a Kubernetes _Job_ running
+on the scan target container image. The image is scanned using the
+[trivy filesystem](https://aquasecurity.github.io/trivy/v0.49/docs/references/configuration/cli/trivy_filesystem/)
+command inside the workload image.
```

Comment:
```suggestion
command inside the job's container.
```

as it is already stated that the job runs using the scanned image. And it's technically a container at this point.